### PR TITLE
Univariate misfit caluclation

### LINKED ISSUE #816: Expose misfit calculations based on the values in storage
Should expose the misfit for each observation point, not just the entire vector

### PR DESCRIPTION
**Pull request by hnformentin**
**Issue**
Resolves #816

### CODE REVIEW

**Comment by ertomatic** [comment]
Can one of the admins verify this patch?

**Comment by sondreso** [comment]
Add to whitelist

File: tests/storage/__init__.py [hunk 1]
```diff
@@ -85,7 +85,9 @@ def add_blob(data):
     observation.add_attribute("region", "1")
 
     response_definition = repository.add_response_definition(
-        name="response_one", indexes_ref=add_blob([3, 5]), ensemble_name=ensemble.name,
+        name="response_one",
+        indexes_ref=add_blob([3, 5, 8, 9]),
+        ensemble_name=ensemble.name,
     )
     active_ref = add_blob([True, False])
```
Comment (sondreso) [inline tests/storage/__init__.py:92]: This needs to be extended to match the new size of the observation

Comment (sondreso) [reply tests/storage/__init__.py:92]: There are also some further down

Comment (ManInFez) [reply tests/storage/__init__.py:92]: The size of the observation has not changed, only the size of the responses

Comment (sondreso) [reply tests/storage/__init__.py:92]: My bad!

File: tests/storage/test_storage_api.py [hunk 2]
```diff
@@ -90,3 +90,34 @@ def test_observation_attributes(populated_db):
...
+
+def test_single_observation_misfit_calculation(populated_db):
+    # observation
+    values_obs = [10.1, 10.2]
+    stds_obs = [1, 3]
+    data_indexes_obs = [2, 3]
+    # response
+    values_res = [11.1, 11.2, 9.9, 9.3]
+
+    misfit_manual = {
```
Comment (sondreso) [inline tests/storage/test_storage_api.py:103]: Would prefer if this was called misfit_expected, but let's move it to the cleanup issue

**Pull request merged**